react/src/components/atoms/forms/InputRadio/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './style.css';
+
+class InputRadio extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
```

Comment:
Remove the constructor above, then:
```suggestion
  handleChange = (event) => {
```